proc_stat_io.go

```diff
@@ -0,0 +1,58 @@
+package procfs
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+type ProcIO struct {
+	// chars read
```

Comment:
Just a style nit: could you start all comments with capital letters and period at the end? (sentence-style)